scripts/irods/database_upgrade.py

```diff
@@ -225,6 +226,30 @@ def run_update(irods_config, cursor, is_upgrade):
                 =======================================================================
             """))
 
+    elif new_schema_version == 13:
+        # password hashing parameters
+        default_password_hashing_parameters = {
+            "algorithm": "scrypt",
+            "parameters": {
+                "N": 16384,
+                "r": 8,
+                "p": 1,
+                "key_length": 64
+            }
+        }
+        database_connect.execute_sql_statement(
+            cursor, "insert into R_GRID_CONFIGURATION values ('authentication', 'password_hashing_parameters', '{}');".format(
+                json.dumps(default_password_hashing_parameters)))
```

Comment:
These reflect the sysadmin's choice for new passwords.

If a sysadmin changes the parameters, it's fine because existing users still have the old algo and params attached to their password info, correct?